doc/dev/mgmt/tests.md

```diff
@@ -15,178 +14,129 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
-which in turn rests upon on a HTTP recording system ([vcrpy](https://pypi.python.org/pypi/vcrpy))
-that enables tests depending on network interaction
-to be run offline.
+This page is to help you write tests for Azure Python SDK when these tests require Azure HTTP requests. The Azure SDK test framework uses the `azure-devtools` package, which in turn rests upon on a HTTP recording system ([vcrpy](https://pypi.python.org/pypi/vcrpy)) that enables tests dependent on network interaction to be run offline.
 
 In this document, we will describe:
--   How to run the tests offline, using previously recorded HTTP interactions,
-    or online, by authenticating with Azure to record new HTTP interactions
+-   How to run the tests offline (using previously recorded HTTP interactions)
+-   How to run the tests online (by authenticating with Azure to record new HTTP interactions)
 -   How to write new tests using our utility classes
 
 # Getting the tests to run
 
-This section describes how to run the SDK tests,
-by installing dependencies into a virtual environment
-and using a helper script and the [pytest](https://docs.pytest.org/en/latest/) test runner
-to choose which tests to run.
+This section describes how to run the SDK tests, by installing dependencies into a virtual environment and using a helper script and the [pytest](https://docs.pytest.org/en/latest/) test runner to choose which tests to run.
 
 ## Running the tests
 
-Azure SDK tests use [pytest](https://docs.pytest.org/en/latest/) test runner.
-To run all the tests, you can use the following command:
+Azure SDK tests use [pytest](https://docs.pytest.org/en/latest/) test runner. To run all the tests, you can use the following command:
 
 ```Shell
 pytest
 ```
 
-You can provide directories or individual files as positional arguments
-to specify particular tests to run
-rather than running the entire test suite. For example:
+You can provide directories or individual files as positional arguments to specify particular tests to run rather than running the entire test suite. For example:
 ```Shell
-pytest -s sdk/storage/azure-mgmt-storage/
+pytest sdk/storage/azure-mgmt-storage/
 pytest sdk/storage/azure-mgmt-storage/tests/test_mgmt_storage.py
 ```
 
-By default, tests run in playback mode,
-using recordings of HTTP interactions to simulate
-requests made against Azure and allow the tests to run offline.
-To run the tests in live (or "recording") mode,
-you'll need to set up token-based Azure authentication.
+If you have print statements in your tests for debugging you can add the `-s` flag to send those print statements to standard output:
+```Shell
+pytest sdk/storage/azure-mgmt-storage/ -s
+```
+
+By default, tests run in playback mode, using recordings of HTTP interactions to simulate requests made against Azure and allow the tests to run offline. To run the tests in live (or "recording") mode, you'll need to set up token-based Azure authentication.
```

Comment:
since you've brought up recording twice by this point, I think it might be a good idea to briefly explain before the "how to run tests in playback" what recording / running playback means